R/rlistings_methods.R

```diff
@@ -14,33 +13,36 @@ dflt_courier <- font_spec("Courier", 9, 1)
 #'
 #' @export
 #' @name listing_methods
-print.listing_df <- function(x, widths = NULL, tf_wrap = FALSE, max_width = NULL, fontspec = NULL, col_gap = 3L,  ...) {
-  tryCatch({
-    cat(
-      toString(
-        matrix_form(x, fontspec = fontspec, col_gap = col_gap),
-        widths = widths,
-        tf_wrap = tf_wrap,
-        max_width = max_width,
-        fontspec = fontspec,
-        col_gap = col_gap,
-        ...
+print.listing_df <- function(x, widths = NULL, tf_wrap = FALSE, max_width = NULL, fontspec = NULL, col_gap = 3L, ...) {
+  tryCatch(
+    {
+      cat(
+        toString(
+          matrix_form(x, fontspec = fontspec, col_gap = col_gap),
+          widths = widths,
+          tf_wrap = tf_wrap,
+          max_width = max_width,
+          fontspec = fontspec,
+          col_gap = col_gap,
+          ...
+        )
       )
-    )
-  }, error = function(e) {
-    if (nrow(x) == 0) {
-      print("No observation in the listing object.")
-    } else {
-      stop(e)
+    },
+    error = function(e) {
+      if (nrow(x) == 0) {
+        print("No observation in the listing object.")
```

Comment:
We should catch it into `rlistings::toString` before computing

Reply:
you mean at line 20, try catch and save it to a variable?

Reply:
btw, i think these lines were just restyled

Reply:
I know but I think we need to change this! Let me work on it tomorrow ;)

Reply:
i see you opened the previous closed issue on this. we can work on a separate PR for this. no problem

Reply:
yes I will add a new issue